### PR TITLE
kava-10 v0.17 update

### PR DESCRIPTION
**Pull request by tasinco**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,38 +3,72 @@ module github.com/figment-networks/kava-worker
 go 1.18
 
 require (
-	github.com/cosmos/cosmos-sdk v0.44.5
+	github.com/cosmos/cosmos-sdk v0.45.4
```
Comment (tasinco) [inline go.mod:6]: FYI: https://github.com/Kava-Labs/kava/compare/v0.16.0...v0.17.0#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R7
The cosmos-sdk was changed.

File: .vscode/launch.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
             "request": "launch",
             "program": "${workspaceRoot}/cmd/worker-kava",
             "env": {
-                "CHAIN_ID": "kava-9",
+                "CHAIN_ID": "kava_2222-10",
```
Comment (tasinco) [inline .vscode/launch.json:9]: The chainID changes https://gateway.pinata.cloud/ipfs/Qmf2Fk2vifniRDBXqnY1c1GSPvkqTbMCRREEWoJV9GPHCH

**Review by christophermanning — COMMENTED** [review]
The helm directory can be changed to just: `helm/kava-worker/...`

Otherwise, these changes look good so far.

File: .github/workflows/kava-worker.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-name: Deploy kava-9
+name: Deploy kava-10
```
Comment (christophermanning) [inline .github/workflows/kava-worker.yml:1]:
```suggestion
name: Deploy kava-worker
```

**Review by christophermanning — APPROVED** [review]

File: go.mod [hunk 4]
```diff
@@ -3,10 +3,20 @@ module github.com/figment-networks/kava-worker
 go 1.18
 
 replace (
```
Comment (tasinco) [inline go.mod:5]: taken from https://github.com/Kava-Labs/kava/blob/f472d119ccd046ce10d7cfbfaab6b0978b5fe980/go.mod#L157

**Pull request merged**